{"status": "broken", "name": "Returning a pattern comprehension with label predicate", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "skip"}, {"name": "feature", "value": "PatternComprehension"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython2"}], "stop": 1578486115474, "historyId": "d1f9a358e9cc9c12ef5b87315f05af21", "start": 1578486115249, "steps": [{"status": "passed", "start": 1578486115250, "stop": 1578486115465, "name": "Given an empty graph"}, {"status": "passed", "start": 1578486115466, "stop": 1578486115470, "name": "And having executed", "attachments": [{"source": "68f1870a-3aae-4563-a5cd-92b769e25f1e-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "passed", "start": 1578486115470, "stop": 1578486115472, "name": "When executing query", "attachments": [{"source": "2ad4dab4-85d6-4625-a8b8-101a82ebba4f-attachment.txt", "type": "text/plain", "name": ".text"}]}, {"status": "broken", "attachments": [{"source": "482c9e49-73a5-4837-aaae-2d20094b787b-attachment.csv", "type": "text/csv", "name": ".table"}], "stop": 1578486115474, "start": 1578486115472, "statusDetails": {"message": "ResponseError: RedisGraph does not currently support pattern comprehension\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "name": "Then the result should be"}, {"status": "skipped", "start": 1578486115474, "stop": 1578486115474, "name": "And no side effects"}], "statusDetails": {"message": "ResponseError: RedisGraph does not currently support pattern comprehension\n", "trace": "  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/laptop-linux/.local/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/Steps.py\", line 84, in step_impl\n    raise exception\n"}, "uuid": "b1b872c7-f8c2-4da5-8bf2-1fcea80373d6"}